### ISSUE TITLE
Forkable base/test Chapter repository

### ISSUE THREAD

**Issue by bnb** [nodeschool/admin#26]
At present, to use Chapter Mode, your terminal needs to be pointed to a directory that has a remote Chapter git repository. The way to set up a Chapter repository at present is to use `$ nodeschool-admin` and clone an existing chapter. What if you don't know a chapter? Pressing enter gives an error, entering `none` gives an error, and entering `default` gives an error.

I would normally suggest that we add an option to create an interface to generate a chapter from scratch. However, there's a much simpler solution that can be used until that can be put in place: have a default chapter repository. I suggest pressing enter or typing `default` in the clone option copy a fresh, empty chapter.

I think this should go in a new repository in the NodeSchool org.

Thoughts?

**Comment by martinheidegger** [comment]
The "purpose" I saw in `nodeschool-admin clone` is to make it easier for already-chapter-maintainers to download and maintain their chapter. The maintainers usually know their chapter which is why i didn't mind to add a selection. `nodeschool-admin list` should show just about every chapter that actively exists. Creating a new chapter has been and still should be an request/approval process (write an issue in [`nodeschool/organizers`](https://github.com/nodeschool/organizers/issues) and get approved). The repo is created by an admin (nodeschoolbot?!) could already contain a good clone?! We could show a list of repos (not chapters) when the selected chapter string is empty. Would that serve your issue?

**Comment by bnb** [comment]
Yes, that would serve the issue of creating a new chapter from a blank slate. Would that also be adaptable to development of `nodeschool-admin`, giving a blank/placeholder chapter to test/work with?

**Comment by fforres** [comment]
I dont think a blank chapter would be much help, considering you have to go through a different process to create a chapter
https://github.com/nodeschool/organizers/blob/master/README.md#how-to-start-a-new-nodeschool-chapter 

Then the application is just serve the purpose of maintaining your existing chapter.
